react/react1/week2/todo-list/src/Components/ToDo/ToDoList.js

```diff
@@ -0,0 +1,21 @@
+import './ToDoList.css';
+
+const ToDoList = (props) => {
+    const todos = props.todos;
+    const title = props.title;
+    const deleteHandler = props.deleteHandler;
```

Comment:
Remember how we used object destructuring:

```javascript
const {todos, title, deleteHandler} = props;
```

Reply:
Yeah!! I was just thinking to document this option too. Just for reference in future. :)